teller.rb

```diff
@@ -2,8 +2,8 @@
 
 BILL_DENOMS = [100,50,20,10,5,1]
 COIN_DENOMS = [50,25,10,5,1]
-BILL_NAMES = {100=>"Hundreds",50=>"Fiftys",20=>"Twentys",10=>"Tens",5=>"Fives",1=>"Ones"}
-COIN_NAMES = {50=>"Half Dollar",25=>"Quarter",10=>"Dimes",5=>"Nickels",1=>"Pennys"}
+BILL_NAMES  = {100=>"Hundreds",50=>"Fiftys",20=>"Twentys",10=>"Tens",5=>"Fives",1=>"Ones"}
+COIN_NAMES  = {50=>"Half Dollar",25=>"Quarter",10=>"Dimes",5=>"Nickels",1=>"Pennys"}
 
 class Teller
```

Comment:
It seems like the teller should:
- get the customer name/id from the menu system (kyosk)
- validate that the user has an account
- act on the user's requests
- close the user interaction and prepare for the next customer
The bank data object should:
- return the customer's balance
- record deposits and withdrawals
- not allow overdraft
- not talk to the user; that's what the customer kyosk is for and that is at the teller's command